.circleci/config.yml

```diff
@@ -20,6 +22,13 @@ jobs:
       - run:
           name: Generate test coverage results
           command:  make build coverage
+      - deploy:
+          name: Build Windows 1.6.8 Binaries
+          command: |
+              make build-windows-k8s K8S_VERSION=1.6.8
+          name: Build Windows 1.7.2 Binaries
+          command: |
+              make build-windows-k8s K8S_VERSION=1.7.2
```

Comment:
This is a temporary placement--we'll want to these builds programmatically via the Circle CI API.